hoomd/Analyzer.h

```diff
@@ -21,6 +21,7 @@
 #include "SharedSignal.h"
 #include "SystemDefinition.h"
 
+#include <limits>
```

Comment:
Does the compile error for the missing `limits` header occur in `Analyzer.h`? I don't see where it would.

Please move the include to only those `.cc` and `.h` files that use symbols from `limits` to reduce compilation time.

Reply:
Done. Sorry I looked for a root file thinking that was the right choice.